src/operator/tensor/indexing_op-inl.cuh

```diff
@@ -167,7 +167,7 @@ AddTakeGradLargeBatchWorkspaceSize(size_t num_keys) {
   size_t exclusivesum_bytes = 0;
   cub::DeviceScan::ExclusiveSum<IndexType*, IndexType*>(NULL, exclusivesum_bytes,
     NULL, NULL, num_keys);
-  size_t temporary_bytes = max(encode_bytes, exclusivesum_bytes);
+  size_t temporary_bytes = std::max(encode_bytes, exclusivesum_bytes);

```

Comment:
(which is probably why someone previously had turned this into a "manual" max function a < b ? b : a)